README.md

```diff
@@ -38,14 +38,14 @@ To run the C++ test suite do (all tests should pass):
 
 To run the Python end-to-end test suite:
 
-start at the top level graphit directory cloned from Github, NOT the build directory
+start from inside the build directory
+
 (All tests would pass, but some would generate error messages from the g++ compiler. This is expected.)
 Currently the project supports Python 2.x and not Python 3.x (the print syntax is different)
 
 ```
-    cd graphit/test/python
-    python test.py
-    python test_with_schedules.py
+    python python_tests/test.py
```

Comment:
Could we say start from the root directory, and show the command "cd build" before showing the commands "python python_tests/test.py ...", this way people can get a more direct sense of where the python_tests folder is?